elephant/test/test_spike_train_correlation.py

```diff
@@ -638,5 +641,38 @@ def test_exist_alias(self):
         self.assertEqual(sc.spike_time_tiling_coefficient, sc.sttc)
 
 
+class SpikeTrainTimescaleTestCase(unittest.TestCase):
+
+    def test_timescale_calculation(self):
+        '''
+        Test the timescale generation using an alpha-shaped ISI distribution,
+        see [1, eq. 1.68]. This is equivalent to a homogeneous gamma process
+        with alpha=2 and beta=2*nu where nu is the rate.
+
+        For this process, the autocorrelation function is given by a sum of a
+        delta peak and a (negative) exponential, see [1, eq. 1.69].
+        The exponential decays with \tau_corr = 1 / (4*nu), thus this fixes
+        timescale.
+
+        [1] Lindner, B. (2009). A brief introduction to some simple stochastic
+            processes. Stochastic Methods in Neuroscience, 1.
+        '''
+        nu = 25/pq.s
+        T = 1000*pq.s
+        binsize = 1*pq.ms
+        timescale = 1 / (4*nu)
+        timescale.units = pq.ms
+
+        timescale_num = []
+        for _ in range(50):
+            spikes = homogeneous_gamma_process(2, 2*nu, 0*pq.ms, T)
+            spikes_bin = BinnedSpikeTrain(spikes, binsize)
+            timescale_i = spike_train_timescale(spikes_bin, 10*timescale)
+            timescale_i.units = timescale.units
+            timescale_num.append(timescale_i.magnitude)
+        correct = np.isclose(timescale.magnitude, timescale_num, rtol=1e-1)
+        self.assertTrue(correct.mean() > 0.9)
```

Comment:
I didn't take a closer look in the first place,
this check is overcomplicated. do you assume it should be the same for 90% of cases?
please use `assert_array_almost_equal`

Reply:
tried to address, please check